README.md

```diff
@@ -31,42 +31,72 @@
 
 ## Introduction ##
 
-dlisio is an LGPL licensed library for working with well logs in Digital Log
-Interchange Standard (DLIS V1), also known as
-[RP66 V1](http://w3.energistics.org/rp66/v1/Toc/main.html). DLIS V2 is
-out-of-scope for this project, as it is quite different and hardly in use in
-the industry. It is an attempt at a powerful community-driven, portable,
-easy-to-use and flexible library for well logs, that can be used to build a
-wide array of applications.
-
-As of version 0.3.0, dlisio is extended to also read Log Information Standard
-79 ([LIS79](http://w3.energistics.org/LIS/lis-79.pdf)). An extended version of
-the LIS79 standard called LIS84/Enhanced LIS exists, but this version is
-currently not supported by dlisio.
-
-Features are added as they are needed; suggestions, defect reports, and
-contributions of all kinds are very welcome.
+dlisio is an LGPL licensed library for reading well logs in Digital Log
+Interchange Standard (DLIS V1), also known as [RP66
+V1](http://w3.energistics.org/rp66/v1/Toc/main.html) and Log Information
+Standard 79 ([LIS79](http://w3.energistics.org/LIS/lis-79.pdf)).
+
+dlisio is designed as a general purpose library for reading well logs in a
+simple, and easy-to-use manner. Its main focus is making all the data and
+metadata accessible while putting little assumptions on how the data is to be
+used. This makes it suitable as a building block for higher level applications
+as well as being used directly.
```

Comment:
Then do at least
as well as (for) being used directly
?
because `suitable as a block`, but you can't say `suitable as being used`. `suitable for being used` makes more sense